respec/template.html

```diff
@@ -436,13 +434,101 @@ <h2>Conformance</h2>
+    </p>
+    <p>
+    <strong>[Dublin Core]</strong> DCMI Metadata Terms, Issued 2020-01-20 by the Dublin Core™ Metadata Initiative.
+    Available at <a href="https://www.dublincore.org/specifications/dublin-core/dcmi-terms/.">https://www.dublincore.org/specifications/dublin-core/dcmi-terms/.</a>
+  </p>
+  <p><strong>[JSON-LD]</strong> JSON-LD 1.1, Gregg Kellogg, Markus Lanthaler, Editors. W3C Recommendation 11 January 2018.
+    Available at <a href="https://www.w3.org/TR/json-ld11/.">https://www.w3.org/TR/json-ld11/.</a>
+  </p>
+  <p><strong>[ODRL]</strong> Open Digital Rights Language (ODRL) Version 2.2, W3C Recommendation 15 September 2016.
+    Available at <a href="https://www.w3.org/TR/odrl/.">https://www.w3.org/TR/odrl/.</a>
+  </p>
+  <p><strong>[OWL]</strong> OWL 2 Web Ontology Language Quick Reference Guide (Second Edition), W3C
+    Recommendation 11 December 2012. Available at <a href="http://www.w3.org/TR/2012/RECowl2-quick-reference-20121211/.">http://www.w3.org/TR/2012/RECowl2-quick-reference-20121211/.</a>
+  </p>
+  <p><strong>[PROV]</strong> PROV-Overview: An Overview of the PROV Family of Documents, W3C Recommendation 30 April
+    2013. Available at <a href="http://www.w3.org/TR/prov-overview/.">http://www.w3.org/TR/prov-overview/.</a>
+  </p>
+  <p><strong>[RDF Concepts]</strong> RDF 1.1 Concepts and Abstract Syntax. Richard Cyganiak, David Wood and Markus
+    Lanthaler, Editors. W3C Recommendation, 25 February 2014. Available at
+    <a href="http://www.w3.org/TR/rdf11-concepts/">http://www.w3.org/TR/rdf11-concepts/</a>
+  </p>
+  <p><strong>[RDF Schema]</strong> RDF Schema 1.1. Dan Brickley and R.V. Guha, Editors. W3C Recommendation, 25
+    February 2014. Available at http:// www.w3.org/TR/rdf-schema/.
+  </p>
+  <p><strong>[SHACL]</strong> Shapes Constraint Language (SHACL), Dimitris Kontokostas, Holger Knublauch, et al., Editors.
+    W3C Recommendation 20 July 2017. Available at <a href="https://www.w3.org/TR/shacl/.">https://www.w3.org/TR/shacl/.</a>
+  </p>
+  <p><strong>[Turtle]</strong> Turtle - Terse RDF Triple Language. Eric Prudhommeaux and Gavin Carothers, Editors. W3C
+    Recommendation 14 January 2014. Available at <a href="http://www.w3.org/TR/turtle/.">http://www.w3.org/TR/turtle/.</a>
+  </p>
+  <p><strong>[W3C Datatypes in RDF and OWL]</strong> XML Schema Datatypes in RDF and OWL, W3C Working Group Note 14 March
+    2006, Available at <a href="http://www.w3.org/TR/2006/NOTE-swbp-xsch-datatypes20060314/.">http://www.w3.org/TR/2006/NOTE-swbp-xsch-datatypes20060314/.</a>
+  </p>
+  <p><strong>[XML Schema Datatypes]</strong> XML Schema Part 2: Datatypes Second Edition. W3C Recommendation 28 October
+    2004. Available at <a href="http://www.w3.org/TR/xmlschema-2/.">http://www.w3.org/TR/xmlschema-2/.</a>
+  </p>
+</section>
+
 <section>
   <h2>Terms and Definitions</h2>
   <p>
     All terms introduced in this specification are given definitions in Chapter 5.
   </p>
 </section>
 
+<section>
```

Comment:
Should be at the bottom, these new sections will also change chapter numbers

Reply:
I changed the chapter 5, don't understand the "bottom" comment

Reply:
Any section you add above the main one (the DPROD Model chapter 5) will increase the chapter numbers of all subsequent sections, that's one other reason to move sections like "Reference" to the bottom (or better: to the appendix, see respec docs)